modules/topicsFpdModule.js

```diff
@@ -73,6 +115,145 @@ export function processFpd(config, {global}, {data = topicsData} = {}) {
   });
 }
 
+/**
+ * function to fetch the cached topic data from storage for bidders and return it
+ */
+export function getCachedTopics() {
+  let cachedTopicData = [];
+  if (!HAS_GDPR_CONSENT || !HAS_DEVICE_ACCESS) {
+    return cachedTopicData;
+  }
+  const topics = config.getConfig('userSync.topics') || bidderIframeList;
+  const bidderList = topics.bidders || [];
+  let storedSegments = new Map(safeJSONParse(coreStorage.getDataFromLocalStorage(topicStorageName)));
+  storedSegments && storedSegments.forEach((value, cachedBidder) => {
+    // Check bidder exist in config for cached bidder data and then only retrieve the cached data
+    let isBidderConfigured = bidderList.some(({bidder}) => cachedBidder == bidder)
```

Comment:
```suggestion
    let isBidderConfigured = bidderList.find(({bidder}) => cachedBidder == bidder)
```
to access `isBidderConfigured?.expiry` below.

Should probably rename `isBidderConfigured` to reflect the type change to object.

Reply:
Variable name isBidderConfigured is replaced **bidderConfigObj**.